libs/designsystem/src/lib/components/slide-button/slide-button.component.scss

```diff
@@ -1,24 +1,45 @@
+@use 'sass:color';
+
+@use '~@kirbydesign/core/src/scss/interaction-state';
 @use '~@kirbydesign/core/src/scss/utils';
-@use 'slide-button.component.shared';
+
+$slide-button-container-width: 256px;
+$slide-button-spacing: 2px;
+$slide-button-width: 52px;
+$slide-button-container-height: $slide-button-width + 2 * $slide-button-spacing;
+$slide-button-container-bg-color: utils.get-color('primary');
+$slider-thumb-bg-color: utils.get-color('white');
+$slide-button-text-color: utils.get-color('primary-contrast');
+$total-slider-thumb-width: $slide-button-width + $slide-button-spacing * 2;
+$slide-button-container-radius: $slide-button-width * 0.5;
+$slide-button-text-padding: 0 $slide-button-container-radius 0 $total-slider-thumb-width;
 
 @mixin _slider-thumb() {
+  @include interaction-state.transition;
+  // This actually doesn't use the state layer and the custom properties - only the mixin "shell"
```

Comment:
This comment is a bit vague and i fear it will lose it's meaning over time. What is meant by "shell"? Why does it only use the mixin shell?

Reply:
The shell is the part of the mixin that wraps everything else (another mixin). I think it's hard to describe without too many code specific details, but if have any suggestions let me know 🙂

The point of the comment is to make it explicit that it doesn't use the state layer or the custom properties even though you apply the same transition as in other components that _do_ use the state layer.

Another option is to leave out the comment, but then there's a risk that readers won't catch the fact. Maybe that's ok? I don't know 🙂

Let me know if you think it makes more sense to leave the comment out and not be explicit about the details.